astroquery/mast/cutouts.py

```diff
@@ -152,6 +161,11 @@ def get_sectors(self, *, coordinates=None, radius=0*u.deg, objectname=None, movi
 
         if moving_target:
 
+            # The Moving Targets service is currently only available for SPOC
+            if product.upper() != "SPOC":
+                product = "SPOC"
+                warnings.warn("Only SPOC is available for moving targets queries. Defaulting to SPOC.", InputWarning)
```

Comment:
I mean, since it should be unlikely for a user to end up here accidentally, we are better off telling them they've gone wrong and won't get the expected result, rather than changing their input options and returning a (probably unexpected) result

Reply:
Fair point, yes I can make this an error instead